IRCBot/modules/search.py

```diff
@@ -0,0 +1,62 @@
+from template import template
+
+import logging
+
+import duckduckgo, google
+import requests, bs4
+
+class raw(template):
```

Comment:
We just need to rename the class from **raw** to **search** and we should be good anticore!